Extending/Backoffice-UI-API-Documentation/index-v10.md

```diff
@@ -0,0 +1,23 @@
+---
+versionFrom: 10.0.0
+---
+
+# Backoffice UI API Documentation
+
+A library of API Reference documentation is auto-generated from the comments within the Umbraco Source Code.
+
+## [Backoffice UI](https://apidocs.umbraco.com/v10/ui)
+
+Angular, JavaScript, CSS & Less UI API references for building Umbraco backoffice components. Note that V9 is using minified versions of the following files:
+
+- The umbraco.directives
+- The umbraco.services
+- The umbraco.resources
+
+__Note: opens a documentation browser that is different from the documentation section you're viewing now.__
+
+
+## [UI Examples](https://our.umbraco.com/packages/developer-tools/ui-examples/)
+There is a package available called 'UI Examples'. This package adds a new 'UI Examples' section to your Umbraco instance with working examples of how to customise the backoffice. 
+
+__Note: opens a documentation browser that is different from the documentation section you're viewing now.__
```

Comment:
:::note
Opens a documentation browser that is different from the documentation section you're viewing now.
:::